components/canvas/webgl_thread.rs

```diff
@@ -630,6 +649,23 @@ impl WebGLThread {
         let _ = completed_sender.send(());
     }
 
+    fn handle_rebuild_svg(&mut self, context_id: WebGLContextId, svg_string: String){
```

Comment:
@jdm 

This function here is being triggered with the correct `svg_string` but it doesn't seem to actually draw to the context and I'm not entirely sure why. Do you have any ideas?